crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -2553,18 +2574,27 @@ pub struct AstPath(#[turbo_tasks(trace_ignore)] Vec<AstParentKind>);
 pub static TURBOPACK_HELPER: &str = "__turbopackHelper";
 
 pub fn is_turbopack_helper_import(import: &ImportDecl) -> bool {
-    import.asserts.as_ref().map_or(true, |asserts| {
+    import.asserts.as_ref().map_or(false, |asserts| {
         asserts.props.iter().any(|assert| {
             assert
                 .as_prop()
                 .and_then(|prop| prop.as_key_value())
                 .and_then(|kv| kv.key.as_ident())
-                .map_or(true, |ident| &*ident.sym != TURBOPACK_HELPER)
+                .map_or(false, |ident| &*ident.sym == TURBOPACK_HELPER)
```

Comment:
Was this broken before, or was it used some other way?

Reply:
It was broken before